static/app/views/issueDetails/streamline/eventSearch.tsx

```diff
@@ -2,15 +2,14 @@ import {useCallback, useMemo} from 'react';
 import orderBy from 'lodash/orderBy';
 
 import {useFetchIssueTags} from 'sentry/actionCreators/group';
-import {fetchTagValues} from 'sentry/actionCreators/tags';
+import {Dataset, fetchTagValues} from 'sentry/actionCreators/tags';
```

Comment:
why are we using a different Dataset? Are there missing fields in the ones from alerts?